### PR TITLE
Removed recursive copy, files are written to output folder on-the-fly…

### PR DESCRIPTION
**Pull request by mXaln**
…. Fixed bug when chunk that has just one verse, deleted immediately after conversion

### CODE REVIEW

File: src/main/kotlin/com/matthewrussell/trchunkbrowser/domain/ConvertDirectory.kt [hunk 1]
```diff
@@ -8,29 +8,26 @@ class ConvertDirectory(private val root: File) {
...
             .doOnError {
                 outDir.deleteRecursively()
             }
     }
 
-    private fun convertInPlace(dir: File) {
+    private fun convertInPlace(dir: File, out: File) {
         val contents = dir.listFiles().toList()
         val wavFiles = contents.filter {
             it.isFile && (it.extension == "wav" || it.extension == "WAV")
```
Comment (KJoslyn) [inline src/main/kotlin/com/matthewrussell/trchunkbrowser/domain/ConvertDirectory.kt]: I probably shouldn't comment on this since it's not part of the PR changes, but this could be it.extension.toLowerCase() == "wav"

File: src/main/kotlin/com/matthewrussell/trchunkbrowser/domain/ConvertDirectory.kt [hunk 2]
```diff
@@ -8,26 +8,31 @@ class ConvertDirectory(private val inputDir: File) {
...
-        for (sub in subDirs) {
-            export(sub, File(outputDir, sub.name))
+    private fun convertDirectory(inputDir: File, outputDir: File) {
+        inputDir.listFiles()?.let {
+            for (file in it.toList()) {
+                if (file.isDirectory) {
+                    convertDirectory(file, File(outputDir, file.name))
+                }
+                else {
+                    if(file.extension == "wav" || file.extension == "WAV") {
```
Comment (KJoslyn) [inline src/main/kotlin/com/matthewrussell/trchunkbrowser/domain/ConvertDirectory.kt]: file.extension.toLowerCase() == "wav"

Comment (mXaln) [reply src/main/kotlin/com/matthewrussell/trchunkbrowser/domain/ConvertDirectory.kt]: Good point, I did it in one of the following PRs

Comment (KJoslyn) [reply src/main/kotlin/com/matthewrussell/trchunkbrowser/domain/ConvertDirectory.kt]: Cool, I saw that later
